README.md

```diff
@@ -278,6 +278,7 @@ default configuration values (and structure of the configuration table) are:
     },
     depth = 1, -- Git clone depth
     clone_timeout = 60, -- Timeout, in seconds, for git clones
+    default_url_format = 'https://github.com/%s' -- Lua format string used for "aaa/bbb" style plugins
```

Comment:
This is the most minor of nits, but this looks more like changing the `default_url` rather than anything about it's format specifically i.e. is the name `default_url` more applicable. Tbh this is a 0.05 % valid nit so feel free to ignore (PS: I know plug does the same but it feels incorrect there as well, since ultimately what user changes when they change this is the url not the way it's structured 🤷🏾‍♂️ )

Reply:
Fair point - I named it this because, as currently used in `plugin_utils`, it does need to be a format string.